Examples/Swift/ViewController.swift

```diff
@@ -101,7 +101,9 @@ class ViewController: UIViewController, MGLMapViewDelegate {
             self.startMultipleWaypoints()
         }))
         alertController.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
-        present(alertController, animated: true, completion: nil)
+        DispatchQueue.main.async {
```

Comment:
The delay we're trying to address here, is that the delay when tapping on "Start Navigation" or "Default UI"?